library/Businessprocess/Renderer/Renderer.php

```diff
@@ -135,34 +136,42 @@ public function countChildNodes()
      * @param $summary
      * @return BaseHtmlElement
      */
-    public function renderStateBadges($summary)
+    public function renderStateBadges($summary, $totalChildren)
     {
         $elements = [];
 
-        foreach ($summary as $state => $cnt) {
-            if ($cnt === 0
-                || $state === 'OK'
-                || $state === 'UP'
-            ) {
-                continue;
-            }
-
-            $elements[] = Html::tag(
-                'span',
-                [
-                    'class' => [
-                        'badge',
-                        'badge-' . strtolower($state)
-                    ],
-                    // TODO: We should translate this in this module
-                    'title' => mt('monitoring', $state)
+        $itemCount = Html::tag(
+            'span',
+            [
+                'class' => [
+                    'item-count',
                 ],
-                $cnt
-            );
-        }
+                'title' => sprintf('%u %s', $totalChildren, mtp(
+                    'businessprocess',
+                    'Child',
+                    'Children',
+                    $totalChildren,
+                    'businessprocess.nodes'
+                ))
+            ],
+            [
+                $totalChildren,
+                ' ',
+                mtp('node', 'Node', 'Nodes', $totalChildren)
```

Comment:
The first argument is the domain, which should be the module's name. (i.e. `businessprocess`)

Also, please use the terms `Child` and `Children` instead. Best to be seen in the title above, which is redundant then btw. Just use the very same instructions.

Reply:
Why haven't you removed the title? Isn't it redundant?

The amount of children is not part of the translated message. It must be. Translators should be able to move it: `mtp('foo', 'just one', 'many of them (%u)', $count)`

Reply:
✅